### PR TITLE
Fix/play ogg audio on Safari

### LINKED ISSUE #1499: Error trying to reproduce tiles with web sources - IOS
The problem occurs when the user is logged and record a sound with an internet connection. the source is accessible via a web URL.
Unhandled Promise Rejection: NotSupportedError: The operation is not supported.

### PR DESCRIPTION
**Pull request by tomivm**
It works but audio are not cached
close #1499

### CODE REVIEW

File: src/components/Board/Board.container.js [hunk 1]
```diff
@@ -456,10 +460,11 @@ export class BoardContainer extends Component {
     return url;
   }
 
-  playAudio(src) {
-    let audio = new Audio();
+  async playAudio(src) {
+    const safariNeedHelp = IS_BROWSING_FROM_SAFARI && src.endsWith('.ogg');
```
Comment (RodriSanchez1) [inline src/components/Board/Board.container.js]: Good name

**Review by RodriSanchez1 — APPROVED** [review]

**Comment by RodriSanchez1** [comment]
LGTM

**Review by RodriSanchez1 — CHANGES_REQUESTED** [review]
Needs to see how to fix the codacy quality analysis

**Comment by tomivm** [comment]
@RodriSanchez1 Should ignore the files added for support ogv.js. I can't do it from my account. 
@martinbedouret Please can you check it?

**Review by martinbedouret — APPROVED** [review]

**Pull request merged**